index.rst

```diff
@@ -2,28 +2,99 @@
+
+Below we describe how some of the Catcher functionality would be implemented.
+
+Alarms
+------
+
+One of the key features expected from the Catcher was the ability to issue alarms that can be displayed in LOVE.
+One option to support this feature would be to create a custom LOVE producer that can be interacted with using a REST API.
+The alert information would be persisted in the EFD by the `flux`_ script, which would then post a message to the producer of the new alert.
+The producer would then read the alert informatino from the EFD and display it.
 
-The First-Look Analysis and Feedback Functionality (FAFF) working group, identified the need for a service that is capable of executing user-defined operations/computation following user-defined rules. This service has been referred  to as Catcher. In essence, the Catcher is not that different from the Watcher or other event handling off-the-shelf technologies like Kapacitor. This technote explores the idea of using influx2.0 event processing capabilities to as an alternative implementation for the Catcher.
+Process Image Data
+------------------
 
-Add content here
-================
+Another important feature of the Catcher was the capability to process camera image data using the OCPS.
+The OCPS is a CSC that is capable of executing DM-pipeline processes.
+It is going to be key in allowing remote processing of Main Camera data during commissioning and operations.
 
-Add content here.
-See the `reStructuredText Style Guide <https://developer.lsst.io/restructuredtext/style.html>`__ to learn how to create sections, links, images, tables, equations, and more.
+If the Catcher is not a CSC, it won't be able to interact direcly with OCPS to process image data.
+One alternative would be to interact with the OCPS remote backend direcly, instead of going through the CSC.
+
+Generate reports
+----------------
+
+The Catcher should also be able to generate reports, with graphs and performing additional lightweight computation.
+The original idea was to rely on parameterized Jupyter notebooks that would later be converted to webpages and rendered online so users could inspect the results, and use as reference for additional followup.
+
+This could still be achieved using a similar solution or relying on a service like noteburst :sqr:`065` :cite:`SQR-065` and Times Square :sqr:`066` :cite:`SQR-066` to execute parameterized notebooks using a web request and publishing the results afterwards.
+
+
+Service Architecture
+====================
+
+Catcher would be a service built on top of existing influxDB and LSP functionality, with ancillary support from LOVE, the OCPS backend and LSP infrastucture.
+It will mostly rely on `flux`_ scripts and `influxDB task`_ to execute operations based on data streams from the EFD.
```

Comment:
Capital letters here.

If you want you can even say "... from the EFD bucket." to help distinguish Sasquatch (the service) from the EFD (a bucket in InfluxDB)